README.md

```diff
@@ -65,7 +65,7 @@ docker/docker.env.
     make setup_env
 
 You may customize the docker/docker.env as needed for your development needs.
-The docker/docker.env file should NOT be committed version control.
+The docker/docker.env file should NOT be committed to version control.
 
```

Comment:
As long as we're looking at this could we change the language a bit?  The docker.env file is already listed in the .gitignore and will not be tracked by default.

https://github.com/chicagopython/chipy.org/blob/73fb2e307cd39e00c193acfbc2ba0a5a5ffb1c7e/.gitignore#L66-L68

The way this is written now makes me feel like action is needed to prevent git from tracking the file.  

Maybe we could say something like "The docker/docker.env file is already listed in .gitignore and will not be tracked by git."

Reply:
@sophm256 let us know if you agree.  @patrickboland4 thanks for bringing this is up.

Reply:
@elmq0022 Nice catch also! I agree with your wording, but I'd change it slightly to: `The docker/docker.env file is listed in .gitignore and will not be tracked by Git.` (removed the word "already" and capitalized "Git")

How does that sound?

Reply:
That's fine by me.  Thanks.

Reply:
@patrickboland4, Please change your PR to the suggested wording in my comment. Thanks!